opm-core/Installation.rst

```diff
@@ -239,13 +229,16 @@ User interface
 The default user interface is based on the web framework Mojolicious_. You need to install:
 
 * Perl (5.10 or above)
-* Mojolicious (4.63 or above, **less than 5.0**)
+* Mojolicious (4.63 or above, **less than 5.0 !**)
 * Mojolicious::Plugin::I18N (version 0.9)
 * DBD::Pg perl module
 * PostgreSQL (9.3 or above)
 * A CGI/Perl webserver
 
-You can install "Mojolicious" using CPAN or your Linux distribution package system if available. Here is an example with CPAN::
+You can install "Mojolicious" with your Linux distribution package system if old enough
+packages of Mojolicious are available.
```

Comment:
This should be reworded since newer versions of mojolicious are compatible;

Reply:
see above